src/ripple/app/tx/impl/Transactor.cpp

```diff
@@ -2054,7 +2065,10 @@ Transactor::operator()()
         hook::HookStateMap stateMap;
         std::vector<hook::HookResult> weakResults;
 
-        doTSH(false, stateMap, weakResults, proMeta);
+        if (!view().rules().enabled(featureIOUIssuerWeakTSH))
```

Comment:
is this condition supposed to be inverted?

Reply:
This PR changes getTransactionalStakeHolders() to be called only once, but since a second call is still needed when the feature is disabled, it is called here.

I will add a comment.